test/support/mocks/MockDeleteCISPeriodService.scala

```diff
@@ -32,9 +33,10 @@ trait MockDeleteCISPeriodService extends MockFactory {
                              employerRef: String,
                              user: User,
                              deductionPeriod: Month,
-                             result: Either[ServiceError, Unit]): CallHandler4[Int, String, User, Month, Future[Either[ServiceError, Unit]]] = {
-    (mockService.removeCisDeduction(_: Int, _: String, _: User, _: Month))
-      .expects(taxYear, employerRef, user, deductionPeriod)
+                             result: Either[ServiceError, Unit]): CallHandler6[Int, String, User,
```

Comment:
should IncomeTaxUserData be passed as a parameter here?

Reply:
for the controller tests, not that interested in what the data is, so left it out.